python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -220,11 +220,10 @@ def get_openlineage_facets(self) -> OpenLineageFacets:
 
         output_dataset: list[OpenlineageDataset] = [OpenlineageDataset(namespace=None, name=None, facets={})]
         if self.output_table:
-            output_database = create_database(self.output_table.conn_id)
             output_dataset = [
                 OpenlineageDataset(
-                    namespace=output_database.openlineage_dataset_namespace(),
-                    name=output_database.openlineage_dataset_name(table=self.output_table),
+                    namespace=self.output_table.openlineage_dataset_namespace(),
```

Comment:
Does the test for load file need to change after this change?

Reply:
No